docs/manage/_lessons.rst

```diff
@@ -70,22 +70,21 @@ Change Lesson Status
 
 A newly created lesson is *Inactive* by default, meaning that learners will not see it in their **Learn** view. To change the lesson status, follow these steps.
 
-1. Click the **Change** link under the *Status* heading.
-2. Select **Active** or **Inactive**.
-3. Click :guilabel:`SAVE` to confirm, or :guilabel:`CANCEL` to exit without change.
+#. Select the desired lesson from the list in the **Lessons** tab.
+#. Click :guilabel:`OPTIONS` button and select **Edit details**. 
+#. Select **Active** or **Inactive** under the *Status* heading.
+#. Click :guilabel:`SAVE CHANGES` to confirm, or :guilabel:`CANCEL` to exit without change.
 
-  .. figure:: img/change-lesson-status.png
-    :alt: 
 
-Change Lesson Visibility
+Change Lesson Recipients
 """"""""""""""""""""""""
 
-You can change the visibility of the lesson that you set when you initially created it. To reassign the lesson to a different group or the entire class, follow these steps.
+You can change the recipients of the lesson from what you set when you initially created it. To reassign the lesson to a different group or the entire class, follow these steps.
```

Comment:
```suggestion
You can change the recipients of the lesson, reassigning the lesson to a different group or the entire class. Follow these steps:
```

Are there any exceptions when a recipient group of a lesson is somehow locked? For instance, if participants from Class A have participated in Lesson X, can they then be removed as participants?

Reply:
I don't believe there is a way to "lock" the lesson, not sure what purpose would that serve. Even if the coach reassigns the lesson, the progress of learners who viewed or interacted with the resources in it is saved, and the lesson is simply being made available to other learners or groups. 

I try to avoid the `:` and use `.` after the `"... these steps"` all through the docs (somebody, somewhere explained why is that preferable, but can't find it atm). If you find another occurrence with `:`, please change to `.`.

Reply:
I can imagine that `:` "is considered harmful" because of the way that content jumps around on responsive displays. Updated the suggestion!